src/main/java/storm/sqs/SqsQueueSpout.java

```diff
@@ -68,6 +68,7 @@
 	public SqsQueueSpout(String queueUrl, boolean reliable) {
 		this.queueUrl = queueUrl;
 		this.reliable = reliable;
+		this.queue = new LinkedBlockingQueue<>();
```

Comment:
which means I also need to make ```queue``` protected and move the instantiation to ```open()``` right ?